src/test/spec/unified_runner/entity.rs

```diff
@@ -43,28 +45,26 @@ impl ClientEntity {
         observer: Arc<EventHandler>,
         observe_events: Option<Vec<String>>,
         ignore_command_names: Option<Vec<String>>,
+        observe_sensitive_commands: bool,
     ) -> Self {
         Self {
             client,
             observer,
             observe_events,
             ignore_command_names,
+            observe_sensitive_commands,
         }
     }
 
     /// Gets a list of all of the events of the requested event types that occurred on this client.
     /// Ignores any event with a name in the ignore list. Also ignores all configureFailPoint
     /// events.
-    pub fn get_filtered_events(
-        &self,
-        observe_events: &Option<Vec<String>>,
```

Comment:
I don't know why these were being passed as parameters, since the values at the one call site were just referencing the `ClientEntity`'s own state, so I updated this while I was here.